src/Core/Models/Business/SubscriptionUpdate.cs

```diff
@@ -42,7 +43,12 @@ public SeatSubscriptionUpdate(Organization organization, StaticStore.Plan plan,
     {
         _plan = plan;
         _additionalSeats = additionalSeats;
-        _previousSeats = organization.Seats ?? 0;
+        _previousSeats = plan.BitwardenProduct switch
+        {
+            BitwardenProductType.PasswordManager => organization.Seats ?? 0,
+            BitwardenProductType.SecretsManager => organization.SmSeats ?? 0,
+            _ => _previousSeats
```

Comment:
What does the default case do? It assigns the unassigned variable to itself?

Reply:
Resolved